.prettierrc.toml

```diff
@@ -0,0 +1 @@
+trailingComma = "none"
```

Comment:
this I added because _most_ of our ts files are formatted this way. Not sure if we want it, or if we should format all ts files with the standard `prettier` settings?